definitions/ext-dora/definition.yml

```diff
@@ -0,0 +1,41 @@
+domain: EXT
+type: DORA
+synthesis:
+  rules:
+    - identifier: doraName
+      name: doraName
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: DoraMetricsSample
+  tags:
+    doraName:
```

Comment:
this is already the name of the entity so there's no need to create a tag out of it.

Reply:
@naxhh removed and updated 👍🏾